examples/knative/powershell/kn-ps-webhook/test/test-payload.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Here's a catch: your server/handler logic assumes a certain schema (e.g. name) which does not really make it a "generic" webhook server. My proposal would be to construct a proper CloudEvent [1] in your handler and use any valid JSON data (incoming webhook) as the data field in the CloudEvent you construct. This way it's really generic (required input is JSON).

This behavior and requirement (e.g. JSON data) should also be described more clearly in the README section.

[1] Example:
Source: `this.fqdn.hostname` (or configurable via config/ENV)
Timestamp: `time.Now()`
ID: `uuidv4.New()`
Type: `com.vmware.generic.webhook.v0` (configurable via config/ENV)
Subject: (not-set)
Data: `jsonFrom(incoming.Body())`